etc/conf/atTelemetry.yaml

```diff
@@ -333,26 +333,27 @@ telemetry:
 
  ROTPA:
   device: ATPtg
-  Stype: Event
-  topic: currentTarget
-  value: rotPA
+  Stype: Telemetry
+  topic: mountPositions
+  value: skyAngle
   collect_after_event: start_collection_event
+  array: undef
 
  RASTART:
   device: ATPtg
   Stype: Telemetry
   topic: mountPositions
   value: ra
   collect_after_event: start_collection_event
-  array: timestamp_start
+  array: undef
```

Comment:
I noticed a bunch of these changed. Before, was it automatically assuming the 0th element of the array and what you did now will allow one to select it?

Reply:
I had plans to have a method called `timestamp_start` to extract from arrays, but it was never implemented. This was defaulting to select the first element from the array. I now changed to `undef` to make it clearer that it doing nothing special and hence picking up the first element of the array.